python/tank/descriptor/io_descriptor/downloadable.py

```diff
@@ -133,3 +133,13 @@ def _post_download(self, download_path):
         downloaded.
         """
         pass
+
+    def is_purgeable(self):
+        """
+        Typycally returns true when used with default bundle cache location.
+
+        An AppStore type descriptor is typically purgeable as long as the
+        bundle cache root location is the default one thus we're checking
+        the 'self._use_non_default_bundle_cache_root' value.
+        """
+        return not self._use_non_default_bundle_cache_root
```

Comment:
given the double negative here, perhaps it's simpler if the state and member variable is `self._use_default_bundle_cache_root` ??

Reply:
I agree that the double negative is rather ugly for that particular line of code BUT my rationale was to keep everything else the same and not add an inversed condition of something that exists already which I though my be confusing and error prone to introduce a inverse condition with similar name.